source/lmp/pair_deepmd.cpp

```diff
@@ -551,7 +557,7 @@ void PairDeepMD::compute(int eflag, int vflag) {
       (numb_models > 1 && (out_freq > 0 && update->ntimestep % out_freq == 0));
   if (do_ghost) {
     deepmd_compat::InputNlist lmp_list(list->inum, list->ilist, list->numneigh,
-                                       list->firstneigh);
+                                       list->firstneigh, mapping);
```

Comment:
If mapping is only correct for the serial run, we may only pass mapping when the number of ranks is 1 to prevent incorrect results.